vhdbuilder/publish/sig-version-create.sh

```diff
@@ -12,8 +12,14 @@
 [[ -z "${MANAGED_IMAGE_RG_NAME}" ]] && echo "MANAGED_IMAGE_RG_NAME is not set" && exit 1
 [[ -z "${VHD_SOURCE}" ]] && echo "VHD_SOURCE is not set" && exit 1
 [[ -z "${OS_NAME}" ]] && echo "OS_NAME is not set" && exit 1
+[[ "Windows" = ${OS_NAME} ]] && [[ -z "${WINDOWS_VERSION}" ]] && echo "WINDOWS_VERSION is not set" && exit 1
 [[ -z "${HYPERV_GENERATION}" ]] && echo "HYPERV_GENERATION is not set" && exit 1
 
+if [ "Windows" = ${OS_NAME} ]; then
+  # Windows image version in SIG follows the pattern: ${WINDOWS_VERSION}.$(date +"%y%m%d"), e.g. 17763.1457.200909
+  IMAGE_VERSION="${WINDOWS_VERSION}.${IMAGE_VERSION}"
```

Comment:
why not let user provide the full version string 17763.1457.200909? in PIR user has to do the same right?

Reply:
In PIR, image version is generated automatically [here](https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/publish/Marketplace/add-image-version-to-sku.sh#L50). 
I will change the code to let user specify the while image version, since we can obtain this from PIR. And add a TODO to change it to be generated automatically after PIR is not supported.

Reply:
Since images will be published to PIR for reference and we don't have to open building artifacts files, until PIR is deprecated, it's pretty easy to obtain the full version. @AbelHu